src/app/component/ProductReviews/ProductReviews.component.js

```diff
@@ -86,10 +97,11 @@ class ProductReviews extends PureComponent {
 
         const percent = parseFloat(STARS_COUNT * (rating_summary || 0) / PERCENT).toFixed(2);
 
-        if (!rating_summary) return this.renderNoRating();
+        if (!review_count) return this.renderNoRating();
```

Comment:
This seems more logical to me as product can be reviewed by many people having `rating_summary === 0`. Therefore we should show 0 stars for the product. Of course, unless we really wanna show 0 / 5 stars xD